can-type.js

```diff
@@ -45,7 +46,9 @@ function booleanNew(value) {
 var maybeValues = Object.freeze([null, undefined]);
 
 function check(Type, val) {
-	throw new Error('Type value ' + typeof val === "string" ? '"' + val + '"' : val + ' is not of type ' + canReflect.getName(Type) + '.'	);
+	var valueType = typeof val;
+	valueType = valueType.charAt(0).toUpperCase() + valueType.slice(1);
```

Comment:
can you do `canReflect.getName(Type[baseTypeSymbol])` to get the name? If so that might be better.

Reply:
@matthewp `Type` is just a type constructor without symbols in this case, `canReflect.getName(Type[baseTypeSymbol])` returns `undefined`.

Reply:
ok, I guess this is fine then.